tests/e2e/run-scanner-v4.bats

```diff
@@ -306,6 +315,18 @@ verify_scannerV4_matcher_deployed() {
     wait_for_ready_pods "${namespace}" "scanner-v4-matcher" 120
 }
 
+verify_central_scannerV4_env_var_set() {
+    local namespace=${1:-stackrox}
+
+    local central_env_vars="$("${ORCH_CMD}" -n "${namespace}" get deploy/central -o jsonpath="{.spec.template.spec.containers[?(@.name=='central')].env}")"
```

Comment:
There are shellcheck linting issues in this line and in the next one, could you fix them?